README.md

```diff
@@ -1,163 +1,263 @@
-
-Investing the time and effort now will pay huge dividends throughout your
-JavaScript programming career. Knowing how to declare and invoke a function is
-great and necessary, but knowing what's actually going on under the hood during
-the declaration and invocation is exponentially more powerful.
+Did you see the changes? The first one was changing the global variable from
+being defined with `const` to `let`. This allows us to reassign a new value to
+it later on, which leads to the second change. Instead of declaring a new
+version of `currentStatus` with `const` inside `changeStatus()`, we _reassign_ a
+new value to the global variable.
+
+When the variable lookup in `showStatus()` occurs, it will act as it originally
+did. It will look in its own scope, then to the global scope. However, because
+the _global_ `currentStatus` gets reassigned a new value before that lookup even
+occurs, it now outputs the status we expect: `Cat tangled all my yarn! 🙀`
+
+> <details><summary>Seeing these two changes, which fix do you personally think is best?</summary>
+> <p>As previously cautioned, it's best practice to avoid nesting too many things when you can. Always opt for the simplest solution, when it makes sense. </p>
+>
+> <p>For our particular example, what if we just wanted to show the status without changing it first? With the nested fix, that wouldn't be possible at all, as the showStatus() function only exists within changeStatus(). We are forced to change the status before we can show it.</p>
+>
+> <p> Of course, you could nest <i>another</i> function inside changeStatus() that handles updating the status separately. Except now you've created three functions, two nested in one, when you could have just stuck with two separate ones.</p>
+>
+> <p>Nesting gets messy exponentially quickly. When you don't need to, don't!</p>
+> </p></details>
+
+## Conclusion
+
+Lexical scope refers to something's home scope. Much like ourselves, a function
+or variable's lexical scope lives in just one place. We can visit friends,
+cafés, work, school, wherever - but our address will always remain at home.
+Lexical scope is the same. Functions can be invoked wherever possible, but its
+home scope, and therefore its accessible scope chain, will always remain the
+same.
+
+As with all things scope, this too may feel like some nebulous concept. Scope is
+not something you consciously do or write. In a way, it just happens. However,
+where you write things determines how they're scoped and what other parts of
+code they have access to. It is an ever-present concept, always there even when
+you're not thinking about it.
+
+As you move through the rest of this course and further on your programming
+journey, just knowing what all this scope magic is will help immensely.
 
 When a variable contains an unexpected value, understanding the scope chain will
 save you countless hours of painful debugging. When you're wondering where to
```

Comment:
Great example of explaining to adult learners how this lesson is immediately relevant to their work!